src/main.js

```diff
@@ -196,44 +203,37 @@ function createWindow () {
       ]
     },
     {
-      label: 'View',
+      label: '&View',
       submenu: [
         {
-          label:'Zoom In',
-          role: 'zoomin',
-          accelerator: 'CommandOrControl+='
-        },
-        {
-          label:'Zoom Out',
-          role: 'zoomout',
-          accelerator: 'CommandOrControl+-'
-        },
-        {
-          label:'Reset Zoom',
-          role: 'resetzoom',
-          accelerator: 'CommandOrControl+0'
+          label:'Reload',
+          click:  function() {
+            win.reload();
+          },
+          accelerator: config['refresh']
         },
-        {
-          type: 'separator'
+        { role: 'forcereload' },
+        { role: 'toggledevtools' },
+        { type: 'separator' },
+        { role: 'resetzoom' },
+        { 
+          role: 'zoomin',
+          accelerator: 'commandOrcontrol+='
         },
+        { role: 'zoomout' },
+        { type: 'separator' },
+        { role: 'togglefullscreen' },
         {
           label:'Show/Hide',
```

Comment:
Please change `label:'Show/Hide'` to `label:'&Show/Hide'`.

Reply:
Done.